src/models/user.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const userSchema = Schema({
+  userName: {
+    type: String,
+    required: [true, 'Username is required'],
+  },
+  firstName: {
+    type: String,
+    required: [true, 'First name is required'],
+  },
+  lastName: {
+    type: String,
+    required: [true, 'Last name is required'],
+  },
+  email: {
+    type: String,
+    required: [true, 'Email is required'],
+  },
+  address: {
+    type: String,
+    required: [true, 'Address is required'],
+  },
+  password: {
```

Comment:
I forgot to add this in the review 
The name should be `password_hash` or `passwordHash` since we are storing only hashed password in the database.

Reply:
I changed it. Thanks for the feedback.